python/deltalake/table.py

```diff
@@ -453,20 +453,68 @@ def file_uris(
 
     file_uris.__doc__ = ""
 
+    def load_to(self, version: Union[int, str, datetime]) -> None:
+        """
+        Load/time travel a DeltaTable with a specified version number, or a timestamp version of the table. If a
+        string is passed then the argument should be an RFC 3339 and ISO 8601 date and time string format.
+
+        Args:
+            version: the identifier of the version of the DeltaTable to load
+
+        Example:
+            **Use a version number**
+            ```
+            1
+            ```
+
+            **Use a datetime object**
+            ```
+            datetime(2023,1,1)
+            ```
+
+            **Use a datetime in string format**
+            ```
+            "2018-01-26T18:30:09Z"
+            "2018-12-19T16:39:57-08:00"
+            "2018-01-26T18:30:09.453+00:00"
+            ```
```

Comment:
should these examples show the actual function invocaton?

Reply:
Yeah, that would be better